src/app/public/modules/wait/wait.component.spec.ts

```diff
@@ -46,12 +47,12 @@ describe('Wait component', () => {
     fixture.detectChanges();
     expect(el.querySelector('.sky-wait')).not.toBeNull();
     fixture.whenStable().then(() => {
-      expect(fixture.nativeElement).toBeAccessible();
+      expectAsync(fixture.nativeElement).toBeAccessible();
```

Comment:
Does this need to be prefixed with `await`? Might not, but just want to confirm the results are the same with/without it.
https://developer.blackbaud.com/skyux/learn/get-started/advanced/accessibility-unit-tests#insert-the-expectasync-function-and-tobeaccessible-matcher

Reply:
You're right. That was my bad. This is my first time using this new matcher and I didn't realize that. I've fixed this and tested locally by making something fail the test and ensured that the matcher did fail the test.